translations/fr.json

```diff
@@ -279,6 +279,9 @@
         },
         {
           "description": "ExcelMerger merges Excel files"
+        },
+        {
+          "description": "The Best Surprise Redis Desktop Client. Powered by Rust and Tauri."
```

Comment:
Other projects' descriptions are missing here, so I think this would be out of order. I'd suggest just leaving this out (seems like the french docs aren't used yet anyways)